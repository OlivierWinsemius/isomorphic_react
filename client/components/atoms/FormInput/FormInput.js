import React from 'react';

export default ({ label, classes, ...props }) => (
    <div className={classes.inputWrapper}>
        <span className={classes.label}>{label}</span>
        <input className={classes.input} {...props} />
    </div>
);
