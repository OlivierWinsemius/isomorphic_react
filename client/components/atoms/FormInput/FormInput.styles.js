export default () => ({
    label: {
        width: '100%',
        flex: 1,
        padding: 6,
        textAlign: 'right',
    },
    input: {
        flex: 3,
    },
    inputWrapper: {
        display: 'flex',
    },
});
