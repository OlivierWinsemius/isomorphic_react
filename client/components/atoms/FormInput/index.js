import injectSheet from 'react-jss';
import FormInput from './FormInput';
import styles from './FormInput.styles';

export default injectSheet(styles)(FormInput);
