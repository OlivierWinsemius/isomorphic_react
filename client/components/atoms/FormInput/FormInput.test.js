import FormInput from './index';
import { shallow } from 'enzyme';

describe('<FormInput />', () => {
    it('matches snapshot', () => {
        const Component = shallow(FormInput);
        expect(Component).toMatchSnapshot();
    })
});
