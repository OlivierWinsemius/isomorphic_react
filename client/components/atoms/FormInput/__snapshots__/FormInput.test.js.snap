// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FormInput /> matches snapshot 1`] = `
<Fragment>
  <span>
    label
  </span>
  <input />
</Fragment>
`;
